crates/http/src/config.rs

```diff
@@ -26,6 +26,8 @@ pub enum HttpExecutorType {
     Spin,
     /// The component implements the Wagi CGI interface.
     Wagi(WagiTriggerConfig),
+    /// The component implements [`wasi-http`](https://github.com/WebAssembly/wasi-http) interface (experimental, subject to change)
+    Wasi,
```

Comment:
Could you update the doc comment on the `HttpExecutorType`? Maybe remove the second sentence or amend it to mention the new variant.